sdcm/cluster.py

```diff
@@ -3490,16 +3491,22 @@ def execute_cmd(cql_session, entity_type):
 
         return list(regular_table_names - materialized_view_table_names)
 
-    def is_table_has_data(self, session, table_name: str) -> bool:
+    def is_table_has_data(self, session, table_name: str) -> (bool, Optional[Exception]):
         """
-        Return True if `table_name` has data in it
+        Return True if `table_name` has data in it.
+        Checking if a result page with any data is received.
         """
-        current_rows = 0
+        error = None
         try:
-            current_rows = session.execute(f"SELECT * FROM {table_name} LIMIT 1").current_rows
+            query = f"SELECT * FROM {table_name}"
+            statement = SimpleStatement(query, fetch_size=10)
+            for paged_result in session.execute(statement):
```

Comment:
@roydahan , @fruch , i added a usage of paged-query here, (following https://github.com/scylladb/scylladb/issues/12268) since code is already modified.
There are other occurrences of non-paged queries in SCT that can be addressed in a separate PR.

Reply:
did you tested it, and it solved the issue ? now that nemesis in that test case is working ?

Reply:
https://github.com/scylladb/scylla-cluster-tests/pull/5555